test/gradcheck.jl

```diff
@@ -1450,11 +1450,7 @@ end
   @test gradient(x -> findfirst(ismissing, x), [1, missing]) == (nothing,)
   @test gradient(x -> findlast(ismissing, x), [1, missing]) == (nothing,)
   @test gradient(x -> findall(ismissing, x)[1], [1, missing]) == (nothing,)
-end
-
-@testset "nograd" begin
-  f(x) = sum(Zygote.nograd(x) + x)
-  @test gradient(f, randn(5)) == (ones(5),)
+  @test gradient(x -> Zygote.ignore(x -> x*x), 1) == (nothing,)
```

Comment:
The test is failing because `ignore` calls a function without arguments. So the test should be written as:

```julia
gradient(x -> Zygote.ignore(() -> x*x), 1) == (nothing,)
```

Reply:
You may as well make that a github suggestion, and I'll commit it :)

Reply:
```suggestion
  @test gradient(x -> Zygote.ignore(() -> x*x), 1) == (nothing,)
```

Reply:
First time I use the github suggestion feature! Hope it's ok.